### DISCUSSION TITLE
Write into s3 directly

### DISCUSSION THREAD

**Discussion by AparnaPillai46** [allure-framework/allure2#1352] [General Discussion]
Can allure load its result files from an S3 bucket and then write it's generated report files out to another S3 bucket?
Is there a solution available for this today ?
https://gitter.im/allure-framework/allure-core?at=5bea2460bb06d73a99656639

**Comment by Avaddon-tfa** [comment]
At the moment there is no such opportunity in Allure Report. We are looking forward to implementing it someday but it's hardly going to happen soon. Try looking at Allure Ruby actions implementation for hosting reports at Google Storage, it might be helpful in case you decide to make your own implementation.

AFAIK it is implemented in Allure TestOps, please, contact Qameta support for more details on this feature.
